src/editor/index.js

```diff
@@ -190,6 +190,13 @@ Inspector.prototype = {
       this.selectEntity(entity, false);
     });
 
+    Events.on('hidecursor', () => {
+      this.cursor.pause();
```

Comment:
Add `this.selectEntity(null)` here.

Reply:
https://fontawesome.com/icons/hand?f=classic&s=regular
We can add the npm dependency "@fortawesome/free-regular-svg-icons"
my AwesomeIcon component and the 2 css rules, see https://github.com/aframevr/aframe-inspector/pull/707
This way you can use any awesome icons
```js
import { faHand } from '@fortawesome/free-regular-svg-icons';
import { AwesomeIcon } from '../AwesomeIcon';
<AwesomeIcon icon={faHand} />
```

Reply:
I took the liberty to continue it a bit I hope you don't mind.